src/adapters/ActivitiesAdapter.js

```diff
@@ -4,14 +4,14 @@ import WebexAdapter from './WebexAdapter';
 
 /**
  * This is a base class that defines the interface that maps activity data.
- * Developers that want to extend `ActivityAdapter` must implement all of its methods,
+ * Developers that want to extend `ActivitiesAdapter` must implement all of its methods,
  * adhering to the exact parameters and structure of the returned objects.
  */
-export default class ActivityAdapter extends WebexAdapter {
+export default class ActivitiesAdapter extends WebexAdapter {
   /**
    * An activity a person performs in Webex.
    *
-   * @typedef {Object} Activity
+   * @typedef  {Object} Activities
```

Comment:
This is just a singular activity so while the adapter's name `ActivitiesAdapter` works, this should just be an `Activity`.